fireblocks_sdk/sdk.py

```diff
@@ -2184,6 +2184,87 @@ def delete_user_group(self, id: str) -> None:
 
         return self._delete_request(url)
 
+    def get_console_users(self):
+        """
+        Gets all Console Users for your tenant
+        """
+
+        url = "/v1/management/console-users"
+
+        return self._get_request(url)
+
+    def get_api_users(self):
+        """
+        Gets all Api Users for your tenant
+        """
+
+        url = "/v1/management/api-users"
+
+        return self._get_request(url)
+
+    def create_console_user(self, first_name: str, last_name: str, email: str, role: str):
+        """
+        Create Console User for your tenant
+        @param first_name: firstName of the user, example: "Johnny".  Maximum length: 30 chars.
+        @param last_name: lastName of the user. Maximum length: 30 chars.
+        @param email: email of the user, example: "email@example.com"
+        @param role: role of the user, for example: "ADMIN"
+        """
+
+        url = "/v1/management/console-users"
+
+        body = {
+            "firstName": first_name,
+            "lastName": last_name,
+            "email": email,
+            "role": role
+        }
+
+        return self._post_request(url, body)
+
+    def create_api_user(self, name: str, role: str, csr_pem: str, co_signer_setup: Optional[str] = None, co_signer_setup_is_first_user: Optional[bool] = False):
```

Comment:
Please use enums or typing.Literal (in this case we'll have to drop support from versions <= v3.7, which is fine but requires few other updates in the repo)